src/Polytopes/PolyhedralFan/properties.jl

```diff
@@ -76,14 +76,43 @@ function Base.iterate(iter::MaximalConeIterator, index = 1)
 end
 Base.length(iter::MaximalConeIterator) = nmaximal_cones(iter.PF)
 
-# function cones(as::Type{T}, PF::PolyhedralFan, cone_dim::Int) where {T}
-#     rtype = AsTypeIdentitiesC(as)
-#     if (cone_dim < 0)
-#         return nothing
-#     end
-#     rcones = Polymake.fan.cones_of_dim(PF.pm_fan,cone_dim-length(lineality_space(PF)))
-#     return PolyhedronOrConeIterator{rtype}(PF.pm_fan.RAYS,rcones, PF.pm_fan.LINEALITY_SPACE)
-# end
+@doc Markdown.doc"""
+    cones(as::Type{T} = Cone, PF::PolyhedralFan, cone_dim::Int)
+
+Return the cones of `PF` of dimension `cone_dim` as an iterator over the type of
+object given by `as`.
```

Comment:
This reads backwards to me. How about this instead:
```suggestion
Return an iterator over the cones of `PF` of dimension `cone_dim`.
```
I've left out `as` deliberately, as it feels more natural to me to explain it in the next paragraph, which is about `as` anyway.

Reply:
This makes sense to me. The same phrasing had been used for `faces(::Polyhedron)`, so I would like to adjust it there in the same way. I will do that on #747 because it better matches the intention of that branch.